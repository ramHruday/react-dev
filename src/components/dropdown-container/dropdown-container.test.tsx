import { shallow } from 'enzyme';
import React from 'react';

import DropdownContainer from './dropdown-container';

describe('DropdownContainer', () => {
  it('should render DropdownContainer', () => {
    const component = shallow(<DropdownContainer title="" onValueToggle={() => {}} />);
    expect(component).toMatchSnapshot();
  });
});
