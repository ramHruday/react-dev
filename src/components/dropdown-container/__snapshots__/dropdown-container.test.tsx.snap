// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownContainer should render DropdownContainer 1`] = `ShallowWrapper {}`;
